sx-question-print.el

```diff
@@ -166,6 +166,15 @@ replaced with the comment."
   :type 'boolean
   :group 'sx-question-mode)
 
+(defcustom sx-question-mode-answer-sort-function
+  #'sx-answer-higher-score-p
+  "Function used to sort answers in the question buffer."
+  :type '(choice
+          (const :tag "Higher-scoring first" sx-answer-higher-score-p)
+          (const :tag "Older first"          sx-answer-older-p)
```

Comment:
I'd prefer newer first. Helps discoverability of new answers and provides an alternative to `higher-scoring`, since older answers usually have higher score anyway.